test/project_types/node/commands/connect_test.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+require 'project_types/node/test_helper'
+
+module Node
+  module Commands
+    class CreateTest < MiniTest::Test
+      include TestHelpers::Partners
+      include TestHelpers::FakeUI
+
+      def test_can_connect
+        context = ShopifyCli::Context.new
+
+        ShopifyCli::Project.stubs(:has_current?).returns(false)
+        ShopifyCli::Commands::Connect.any_instance.expects(:default_connect)
+          .with('node')
+          .returns('node-app')
+        context.expects(:done)
+          .with(context.message('core.connect.connected', 'node-app'))
```

Comment:
Slight nitpick: if we're using specific versions of the command for node / rails, should we have specific messages for them? Or we could also use the apps namespace that was added to the core messages file instead.

Reply:
Yes, it would be a good idea to split the messages as well!